huxley/www/js/components/ChairDelegateEmailView.js

```diff
@@ -90,29 +92,35 @@ class ChairDelegateEmailView extends React.Component {
   render() {
     return (
       <InnerView>
-        <TextTemplate>{ChairDelegateEmailViewText}</TextTemplate>
+        <TextTemplate style={{width: "50%"}}>{ChairDelegateEmailViewText}</TextTemplate>
         <form>
-          <div className="table-container">
-            <table>
-              <thead>
-                <tr>
-                  <th>Assignment</th>
-                  <th>Email</th>
-                </tr>
-              </thead>
-            </table>
-            <div>
-              <table className="table highlight-cells">
-                <tbody>
-                  {Object.keys(this.state.countries).length > 0 ? (
-                    this.renderEmailRows()
-                  ) : (
-                    <tr />
-                  )}
-                </tbody>
-              </table>
-            </div>
-          </div>
+          <Table
+            emptyMessage="You don't have any delegates in your committee."
+            isEmpty={!this.state.delegates.length}
+          >
+            <thead>
+              <tr>
+                <th>Assignment</th>
+                <th>Email
+                  <Button
```

Comment:
Functionality seems great! This is more stylistic, but can you move the button outside of the table? You should still be able to access the data from the store as you did, but this will remove the empty column that appears since the button is currently a title of a column.